src/Workspaces/Core/Portable/Workspace/ProjectSystem/ProjectSystemProjectFactory.cs

```diff
@@ -107,7 +108,7 @@ public async Task<ProjectSystemProject> CreateAndAddToWorkspaceAsync(string proj
                 name: projectSystemName,
                 assemblyName,
                 language,
-                compilationOutputFilePaths: default, // will be updated when command line is set
+                compilationOutputInfo: new(creationInfo.CompilationOutputAssemblyFilePath), // will be updated when command line is set
```

Comment:
I'm not sure about the comment after, but anecdotally this can be set here. If it gets updated later I believe that's fine?

Reply:
I read the comment as "its okay to be null as teh value is filled in later", but since this fills in the value now, I'd remove the comment.